javalib/src/main/scala/java/io/FilterReader.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.io
+
+abstract class FilterReader protected (protected val in: Reader) extends Reader {
+
+  if (in == null) throw new NullPointerException
```

Comment:
```suggestion
  in.getClass() // null check
```
like for example at
https://github.com/scala-js/scala-js/blob/b8fb2f28aff3dac46e35c101550c732bf0dbe562/javalib/src/main/scala/java/util/Comparator.scala#L42